tests/Form/FormUtilsTestTrait.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Tests\AppBundle\Form;
+
+use Symfony\Component\DomCrawler\Form;
+
+trait FormUtilsTestTrait
```

Comment:
Why not in `ControllerTestTrait` ?

Reply:
I thought it could be used without the tools of the ControllerTestTrait. But I can merge if you think it's overkill.

Reply:
yes please, we'll do it if it's needed ;-)